coursemapper-kg/concept-map/src/graph_db.py

```
@@ -132,7 +174,35 @@ def _save_graph(tx: ManagedTransaction, graph: Graph):
+        
+                )
+            # Step 2: Update all other concept nodes for the same material, setting lastEdited to false.
+            tx.run(
+                    """
+                    MATCH (c:Concept)
+                    WHERE c.mid = $mid AND c.uid <> $cid AND c.lastEdited = true
+                    SET c.lastEdited = false
+                    RETURN c
+                    """,
+                    mid=material_node.id,
+                    cid=node.id
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)